src/app/systelab-components/loading/README.md

```diff
@@ -2,11 +2,32 @@
 
 Component to show a loading wheel.
 
+Sometimes when user submits a form or the application is loading is needed to disable the content of the whole application in order to avoid user interactions with the UI while the process of the submit or loading ends.
```

Comment:
Sometimes, when submitting a form or loading information, content should be disabled as no user interaction is allowed.